dotnet-algorand-sdk/V2/Indexer/LookupApi.cs

```diff
@@ -419,9 +419,6 @@ public System.Threading.Tasks.Task<Response4> TransactionsGetAsync(int? limit, s
         /// <exception cref="ApiException">A server side error occurred.</exception>
         public async System.Threading.Tasks.Task<Response4> TransactionsGetAsync(int? limit, string next, string note_prefix, TxType? tx_type, SigType? sig_type, string txid, ulong? round, ulong? min_round, ulong? max_round, int? asset_id, System.DateTimeOffset? before_time, System.DateTimeOffset? after_time, ulong? currency_greater_than, ulong? currency_less_than, string account_id, bool? rekey_to, System.Threading.CancellationToken cancellationToken)
         {
-            if (account_id == null)
```

Comment:
?

Reply:
this API is to fetch transactions not just for accounts